.github/mergeable.yml

```diff
@@ -0,0 +1,269 @@
+version: 2
+mergeable:
+  # Let's welcome contributors.
+  - when: pull_request.opened
+    name: "Hola!"
+    validate: []
+    filter:
+      - do: author
+        must_exclude:
+          regex: 'dependabot'
+    pass:
+      - do: comment
+        payload:
+          body: >
+            Thanks for contributing towards Open Social! A maintainer from @goalgorilla/maintainers will review your changes shortly. Please don't be discouraged if it takes a while.
+            In the meantime, we have some automated checks running and it might be that you will see me comment with some tips or requests to speed up the review process. :blush:
+        leave_old_comment: true
+
+  # Adds label 'status: needs review' once a PR is ready for review.
+  - when: 'pull_request.ready_for_review'
+    name: 'Status check'
+    pass:
+      - do: labels
+        add: 'status: needs review'
+        delete: 'status: needs work'
+
+  # Check for a valid title.
+  - when: pull_request.*
+    name: "Valid issue title"
+    validate:
+      - do: title
+        begins_with:
+          match: ['Issue #', 'Internal:', 'Updates:', 'Hotfix:']
+    fail:
+      - do: comment
+        payload:
+          body: >
+            In order for us to match your pull request to the correct open source or internal issue, the title should either:
+              - begin with a reference to a Drupal.org issue number in the format of "Issue #issuenumber"
+              - begin with Internal: if it's related to an internal ticket
+              - begin with Updates: if we are updating one of our dependencies
+              - begin with Hotfix: if we are creating a hotfix
+      - do: labels
+        add: 'blocked: drupal.org ticket'
+      - do: checks # default fail case
+        status: 'failure' # Can be: success, failure, neutral, cancelled, timed_out, or action_required
+    pass:
+      - do: labels
+        remove: 'blocked: drupal.org ticket'
+      - do: checks # default fail case
+        status: 'success'
+
+  # Check for meaningful commit messages.
+  - when: pull_request.*
+      name: "Check commit messages"
+      validate:
+        - do: commit
+          message:
+            regex: '^(coding standards|chore|fix|refactor|test|style|perf)(\(\w+\))?:.+$'
+            skip_merge: true
+      fail:
+        - do: comment
+          payload:
+            body: >
+              It would be great if you could create helpful commit messages, maybe a rebase would work before putting it up for review? ;)
```

Comment:
It would be nice if this was a more "helpful/motivational" message, maybe linking towards documentation or some inline comment with documentation if that is not too lengthy.

Reply:
Done